src/material-examples/cdk-datepicker-overview/cdk-datepicker-overview-example.ts

```diff
@@ -0,0 +1,50 @@
+import {Component} from '@angular/core';
+import {CalendarView, DateAdapter} from '@angular/cdk/datetime';
+
+
+/** @title Basic CDK datepicker */
+@Component({
+  selector: 'cdk-datepicker-overview-example',
+  templateUrl: 'cdk-datepicker-overview-example.html',
+  styleUrls: ['cdk-datepicker-overview-example.css'],
+})
+export class CdkDatepickerOverviewExample {
+  _dateSelected() {
+    console.log('Date changed');
```

Comment:
for these examples its better to do something the user can see, even if its just adding the message to a little virtual console made out of a scrolling div. that way people don't have to open their browser console to see what's happening